core/modules/savers/put.js

```diff
@@ -42,14 +42,20 @@ PutSaver.prototype.save = function(text,method,callback) {
 	// Merging would be ideal, and may be possible using future generic merge flow
 	req.onload = function() {
 		if (this.status === 200 || this.status === 201) {
+			this.etag = this.getResponseHeader("ETag");
 			callback(null); // success
 		}
+		else if (this.status === 412) { // edit conflict
+			callback("File changed on server");
```

Comment:
Here's an example of raising a localised error:

https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/saver-handler.js#L81

And here's where that message is defined:

https://github.com/Jermolene/TiddlyWiki5/blob/master/core/language/en-GB/Misc.multids#L52

Just add another key for the new message.